### ISSUE TITLE
Can't run GEE script

### ISSUE THREAD

**Issue by tlambert720** [amydecastro/wildfire_fuel_update_in_GEE#1]
I attempted to copy the code into Google Earth Engine, however the second line presented an error: "Line 5: lulc is not defined"

**Comment by tlambert720** [comment]
Thank you, that helped. There was another error at the next line though: "Line 7: medium_extent is not defined". I'm guessing there were other dependencies you used as well and variables you defined not mentioned here on github.

**Comment by amydecastro** [comment]
Yes, that's true. medium_extent is a domain extent used in a WRF-Fire simulation. 
May I ask what you are using this for? 

This publication may help with your understanding of the workflow in that script: https://www.mdpi.com/2072-4292/14/6/1447

And you've probably found it, but just in case, here is the GEE developer's page to help you with learning GEE: https://developers.google.com/earth-engine

**Comment by amydecastro** [comment]
Congrats on your internship. This code was flung up here rather haphazardly. Please feel free to reach out using the correspondence email address listed on that paper if you have further questions.
